### ISSUE TITLE
x/pkgsite: package build and godoc links/images lack a11y features

### ISSUE THREAD

**Issue by jba** [golang/go#41112]
Package build and godoc links/images lack accessibility features to indicate the semantic content of the image. E.g. Alt text displayed is “Build Status,” but lacks “Passing,” which is the more important information. If possible, use plain text to display build status, or set alt text dynamically.
